po/es.po

```diff
@@ -0,0 +1,706 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE paru'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the paru package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: paru VERSION\n"
+"Report-Msgid-Bugs-To: https://github.com/Morganamilo/paru\n"
+"POT-Creation-Date: 2021-06-28 03:24+0000\n"
+"PO-Revision-Date: 2021-06-28 18:02+0200\n"
+"Language: es\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"X-Generator: Poedit 3.0\n"
+
+#: src/clean.rs:31
+msgid "Do you want to remove ALL AUR packages from cache?"
+msgstr "¿Quieres eliminar TODOS los paquetes de AUR de la caché?"
+
+#: src/clean.rs:33
+msgid "Do you want to remove all other AUR packages from cache?"
+msgstr "¿Quieres eliminar todos los demás paquetes de AUR de la caché?"
+
+#: src/clean.rs:40
+msgid "Clone Directory: {}"
+msgstr "Clonar Directorio: {}"
+
+#: src/clean.rs:46
+msgid ""
+"\n"
+"Diff Directory: {}"
+msgstr ""
+"\n"
+"Directorio de Diffs: {}"
+
+#: src/clean.rs:48
+msgid "Do you want to remove all saved diffs?"
+msgstr "¿Quieres eliminar todos los diffs guardados?"
+
+#: src/clean.rs:64
+msgid "can't open diff dir: {}"
+msgstr "no se puede abrir el dir de diff: {}"
+
+#: src/clean.rs:75 src/clean.rs:133
+msgid "could not remove '{}'"
+msgstr "no se pudo eliminar '{}'"
+
+#: src/clean.rs:93
+msgid "can't open clone dir: {}"
+msgstr "no se puede abrir el directorio clonado: {}"
+
+#: src/command_line.rs:127
+msgid "option {} expects a value"
+msgstr "opción {} espera un valor"
```

Comment:
```suggestion
msgstr "la opción {} requiere un valor"
```

The exact verb to use is largely irrelevant, but I always see "la opción..." in other translations